archey/colors.py

```diff
@@ -3,6 +3,12 @@
 from bisect import bisect
 from enum import Enum
 
+import re
+
+
+# REGEXP compiled pattern matching ANSI/ECMA-48 escape codes.
+ANSI_ECMA_REGEXP = re.compile(r'\x1b\[\d+(?:(?:;\d+)+)?m')
```

Comment:
I've preferred compiled a proper pattern for the whole `colors` module (and fixed matching in case of more-than-two-display-attributes.